spec.bs

```diff
@@ -2264,7 +2264,8 @@ To <dfn>append to a bidding signals per-interest group data map</dfn> given an [
 1. [=map/For each=] |sourceKey| → |sourceValue| of |sourceMap|:
   1. If |igNames| does not [=set/contain=] |sourceKey|, [=iteration/continue=].
   1. Let |perGroupData| be a new [=bidding signals per interest group data=].
-  1. Issue(WICG/turtledove#1144): handle priority vector.
+  1. Issue: handle priority vector

```

Comment:
Let's make it an issue, not a step, because the rendered spec has an [auto-generated section](https://wicg.github.io/turtledove/#issues-index) for all issues.

Reply:
It's both an issue and a step, and it is appearing in that auto-generated section (see the preview link).

I like keeping it as a step since this is the point where we specifically want to handle the priority vector.

Reply:
oh you're right, didn't notice it was an issue as well.